src/main/content/docs.html

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Docs
+js:
+- docs
+permalink: /docs/
+--- 
+
+{% assign docs-version = '' | split: ','  | sort %}
+{% assign docs = site.pages | where: 'layout', 'doc' %}
+
+{% for doc in docs %}
+    {% assign dir = doc.dir | split: '/' %}
+    {% if dir[2] %}
+        {% assign docs-version = docs-version | push: dir[2] | uniq | sort | reverse %}
+    {% endif%}
+{% endfor %}
+
+<script id="docs-version-json" type="application/json">
```

Comment:
this shouldn't be in a script tag the way that it currently is. You should assign it to a constant variable in the `docs.js ` file. Something like:
```
const versions = '{{docs-version}}'
```
might need to use split if its not already an array, then you can loop through the array in js

you shouldn't need the count, you can probably get that in js during your loop